typescript/infra/scripts/utils.ts

```diff
@@ -340,7 +340,7 @@ export async function getRouterConfig(
         ? await multiProvider.getSignerAddress(chain)
         : owners[chain],
       mailbox: core.getContracts(chain).mailbox.address,
-      hook: igp.getContracts(chain).defaultIsmInterchainGasPaymaster.address,
+      hook: igp.getContracts(chain).interchainGasPaymaster.address,
```

Comment:
did you remove other usage of this?

Reply:
defaultIsmInterchainGasPaymaster is removed from all places except the config files.